ksql-engine/src/main/java/io/confluent/ksql/engine/InsertValuesExecutor.java

```diff
@@ -392,7 +389,7 @@ private byte[] serializeKey(
           .serializer()
           .serialize(dataSource.getKafkaTopicName(), keyValue);
     } catch (final Exception e) {
-      throw new KsqlException("Could not serialize key: " + keyValue, e);
+      throw new KsqlException("Could not serialize key: <retracted>", e);
```

Comment:
nit: should we simplify this to `Could not serialize key.` and leave out the redaction entirely, since the redacted value isn't helpful in the error message?